deploy/deploy.go

```diff
@@ -429,6 +451,10 @@ func (e *ExternalSpec) GetDockerNetworkResourceName() string {
 	return e.Network
 }
 
+func (e *ExternalSpec) Apply(data []byte) error {
+	return logCommandWithInput(data, "kubectl", "apply", "-f", "-")
```

Comment:
This would be testable if you used a variable:

```
var kubectl = "kubectl"
```

and then in the test did something like `kubectl = "echo"`

Reply:
ack